tests/conftest.py

```diff
@@ -1,6 +1,5 @@
 import pytest
-from app import create_app
-import app
```

Comment:
Please rerun the tests, seems that they will file

Reply:
fixed